config/routes.rb

```diff
@@ -99,6 +99,7 @@
       end
       resources :restrictions, only: %i[index create] # as multiple restrictions
       resource :about_the_financial_assessment, only: %i[show update]
+      resource :update_email_address, only: %i[show update]
```

Comment:
I would just call this route `email_address` instead of `update_email_address`. It feels more REST/RoR.
And it also fits with the other routes.
For example we have `property_value`, not `update_property_value`,
`applicant`, not `update_applicant`, etc...